client/.storybook/theme.js

```diff
@@ -17,7 +15,7 @@ export default create({
   fontCode: 'monospace',
 
   // Text colors
-  textColor: 'white',
+  textColor: '#ed7d59',
```

Comment:
> * Changes the text colour to something readable ( white looks better in the sidebar, but is unreadable in the menu and addons window )

Is it too hard to make the sidebar text white and this orange?

Reply:
I tried! Didn't want to spend too long - there isn't support in the storybook theme, and I tried adding `GlobalStyles` but even at the `TestingProvider` level I couldn't get that to behave.
So after playing for a while.. I gave up on it 🤷 

buuut.. have checked again. And adding a `manager-head.html` allows you to modify the storybook framework css. So that's now done 😁